docs/readme.md

```diff
@@ -82,15 +82,38 @@ log.setLevel(flags.logLevel);
 launchChromeAndRunLighthouse('https://example.com', flags).then(...);
 ```
 
+## Configuration
+In order to configure Lighthouse programmatically, you need to pass the config object as the 3rd argument.
+
+**Example:**
+```js
+{
+  extends: 'lighthouse:default',
+  settings: {
+    onlyAudits: [
+      'first-meaningful-paint',
+      'speed-index-metric',
+      'estimated-input-latency',
+      'first-interactive',
+      'consistently-interactive',
+    ],
+  },
+}
+```
+
+You can extend base configuration from either [lighthouse:default](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/default-config.js) or [lighthouse:full](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/full-config.js). Alternatively, you can build up your own configuration from scratch to have complete control.
+
+For more information on the types of config you can provide, see [Lighthouse Configuration](https://github.com/GoogleChrome/lighthouse/blob/master/docs/configuration.md).
+
 ## Testing on a site with authentication
 
 When installed globally via `npm i -g lighthouse` or `yarn global add lighthouse`,
 `chrome-debug` is added to your `PATH`. This binary launches a standalone Chrome
 instance with an open debugging port.
 
 1. Run `chrome-debug`. This will log the debugging port of your Chrome instance
-1. Navigate to your site and log in.
-1. In a separate terminal tab, run `lighthouse http://mysite.com --port port-number` using the port number from chrome-debug.
+2. Navigate to your site and log in.
+3. In a separate terminal tab, run `lighthouse http://mysite.com --port port-number` using the port number from chrome-debug.
```

Comment:
these line numberings were intentional (markdown takes care of the actual numbers making changes to the list easier)

Reply:
I think there was an extra space before the first number, which got resolved when I commited.

Reply:
> I think there was an extra space before the first number, which got resolved when I commited.

can you revert these lines? They're meant to be just 1., 1., 1., etc

Reply:
It's showing as 1. 2. 3. on the master version. See:

https://github.com/GoogleChrome/lighthouse/blob/master/docs/readme.md#testing-on-a-site-with-authentication

Or am I missing something?

Reply:
check out the raw version: https://raw.githubusercontent.com/GoogleChrome/lighthouse/master/docs/readme.md

it's the markdown viewer that's makes it rendered as 1. 2. 3.